### PR TITLE
Add Storage Cover

### PR DESCRIPTION
**Pull request by Spicierspace153**
implements storage cover
2 * 8 for holding molds for extruder

### CODE REVIEW

File: src/main/java/com/gregtechceu/gtceu/common/cover/StorageCover.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            public int getSlotLimit(int slot) {
+                return 1;
+            }
+        };
+    }
+
+    @Override
+    public Widget createUIWidget() {
+        final var group = new WidgetGroup(0, 0, 176, 85);
+        for (int slot = 0; slot < 18; slot++) {
```
Comment (screret) [inline src/main/java/com/gregtechceu/gtceu/common/cover/StorageCover.java]: could check inventorySize here instead of hardcoding for 18 slots.

Comment (screret) [reply src/main/java/com/gregtechceu/gtceu/common/cover/StorageCover.java]: also, this method has 2x as many empty lines as it needs.

**Review by screret — CHANGES_REQUESTED** [review]

File: src/main/java/com/gregtechceu/gtceu/common/cover/StorageCover.java (hunk 1 above)
Comment (Spicierspace153) [reply src/main/java/com/gregtechceu/gtceu/common/cover/StorageCover.java]: illl fix this when i add a tab which means burnout has to be over 👍

**Comment by JuiceyBeans** [comment]
~~Doesn't drop the items in the cover~~

**Comment by JuiceyBeans** [comment]
~~Items dont persist after relogging~~

**Review by screret — APPROVED** [review]

**Review by krossgg — APPROVED** [review]

**Pull request merged**